test/util/images/images.go

```diff
@@ -17,29 +20,32 @@ const (
 
 func init() {
 	registry = os.Getenv("IMAGE_REGISTRY")
+	if registry == "" {
+		registry = "quay.io/openshift-kni/"
+	}
+
+	cnfTestsImage = os.Getenv("CNF_TESTS_IMAGE")
+	if cnfTestsImage == "" {
+		cnfTestsImage = "cnf-tests:4.5"
+	}
 
 	images = map[string]imageLocation{
 		TestUtils: {
-			name:    "cnftest-utils",
-			registy: "quay.io/openshift-kni/",
-			version: "4.5",
+			registy: registry,
+			image:   cnfTestsImage,
 		},
 	}
 }
 
 type imageLocation struct {
-	name    string
 	registy string
-	version string
+	image   string
 }
 
-// For returns the image to be used for the given key
-func For(name string) string {
-	img, ok := images[name]
+// TestImage returns the test image to be used
```

Comment:
This can be just `Test` as the package name is already images.

Also, since there's only one image the map is redundant.
You can use directly the `registry` - `cnfTestsImage` variable